### DISCUSSION TITLE
Cannot fork

### DISCUSSION THREAD

**Discussion by james-mccollough** [community/community#29820] [Other Feature Feedback, Questions, & Ideas]
I created a public repository today, and have 0 forks listed for it.  However, when I attempt to create a fork, I get this:

No more forks can be created. These forks already exist:

But it doesn't list any forks.  Is there a setting or flag I need to change?

**Comment by airtower-luna** [comment]
It sounds like you might be trying to create a fork of your own repository? Doing that wouldn't really make sense, you can just create branches in the repository. The point of a fork is usually to have your own copy of someone else's repository.

**Comment by jfullstackdev** [comment]
it's hard to guess here what you are referring to, 

![image](https://user-images.githubusercontent.com/47092464/185281431-5ac7ff4e-04b7-46a9-9d4c-5887124ba45e.png)

**Comment by radar-blip** [comment]
You cant fork your own repository, it wont allow it. Thats why the messages. Unless you have another account of course.
